.azure-pipelines-k8s-matrix.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
           sudo install minikube-linux-amd64 /usr/local/bin/minikube
           sudo minikube start --driver=none --kubernetes-version $(kubernetes_version)
-          # Although the kube and minikube config files are in placed $HOME they are owned by root
+          sudo mv /root/.kube /root/.minikube $HOME
```

Comment:
My only concern is that this won't work as expected. I search around but could not find the equivalent to the github-action for azure.

Reply:
The comment said they are already in `$HOME`. Are they no longer put there, after this minikube install/run is executed on ubuntu 20.04?

Reply:
According to this run I don't think so...

https://github.com/lensapp/lens/runs/3602487897?check_suite_focus=true

Reply:
OK, if it works...